packages/common/vitest.setup.ts

```diff
@@ -1,5 +1,17 @@
 import 'vi-fetch/setup';
 import 'reflect-metadata';
+import { mockService } from './test/mockService';
+import LogService from './src/services/logging/LogService';
+
+beforeEach(() => {
+  mockService(LogService, {
```

Comment:
That is possible, but I'm not sure if this needs to be enabled by default, right?

Reply:
I mean wouldn't it be just as easy to do something like:
```suggestion
  mockService(LogService, new LogService(new ConsoleTransporter()))
```

Reply:
That would probably work, but I do not recommend instantiating Inversify modules manually because it can break when changing injection arguments.

I'm still unsure if we really want the logging output mixed with the Vitest output. If you think this is useful why don't we do this instead:

```ts
const makeLogger = (level: string) => {
  return (scope: string, message: string, extra: unknown) => {
    return import.meta.env.APP_DEBUG && console.log(`${level} [${scope}] ${message}`, extra);
  };
};
 
mockService(LogService, {
  debug: makeLogger('debug'),
  info: makeLogger('info'),
  warn: makeLogger('warn'),
  error: makeLogger('error'),
  fatal: makeLogger('fatal'),
});
```

Reply:
After the latest change, the above becomes:

```ts
mockService(LogTransporter, {
  log: console.log,
});
```

Reply:
That looks good to me. I'm fine if you want to make it conditional with an environment variable, just so long as it's easy enough to enable when someone is stuck trying to debug tests.